src/encoded/static/components/static-pages/components/benchmarking/BenchmarkingTable.js

```diff
@@ -255,6 +255,14 @@ export const BenchmarkingTable = (props) => {
                     'status',
                     'validation_errors.name',
                     'version',
+                    // File facets used in production browse ui
+                    'access_status',
+                    'donors.display_title',
+                    'donors.age',
```

Comment:
The "Donor Age" and "Donor Sex" facets aren't rendering for me. I'm not sure, but embedding
```suggestion
         "file_sets.libraries.analytes.samples.sample_sources.donor.age",
        "file_sets.libraries.analytes.samples.sample_sources.donor.sex",
```
instead of `donors.age` and `donors.sex` may work

Reply:
Oh interesting. Can you send a screenshot of what that's looking like for you? It's appearing like this on wolf for me:
[![Image from Gyazo](https://i.gyazo.com/afb88ebe2b04893c44630d0b338de901.png)](https://gyazo.com/afb88ebe2b04893c44630d0b338de901)

Reply:
<img width="1184" alt="Screenshot 2024-12-19 at 10 07 24 AM" src="https://github.com/user-attachments/assets/99ae5640-0dbc-45df-81ef-11196b607ea1" />
<img width="1169" alt="Screenshot 2024-12-19 at 10 07 44 AM" src="https://github.com/user-attachments/assets/f997795a-1500-4d7e-b02c-8bc00f893040" />
For some reason the facet for Donor Age isn't selecting the files, even though they are all from a 47-year-old donor, and for Donor Sex, no value is showing up